src/components/Radio.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+import COLORS from '../styles/colors';
+
+export const FormRadio = styled.input`
+  border: 1px solid ${COLORS.ijpBlue};
+`;
+
+export const Radio = styled.input``;
```

Comment:
are we planning to use the Radio component anywhere else? if not, should we delete? otherwise, are we planning to add anything to this styled component?

Reply:
I don't think we are, I just forgot to delete this file. Thanks for catching that!